### DISCUSSION TITLE
Export to iNaturalist

### DISCUSSION THREAD

**Discussion by David-Birdbrain** [Mattk70/Chirpity-Electron#140] [Ideas]
**Is your feature request related to a problem? Please describe.**
iNaturalist is a repository for recording animal observations: https://www.inaturalist.org
It has a robust API
Would be great to be able to post detections from Chiperty to iNat as observations

Added notes: 30th Aug
Business driver: I run a reforestation and biodiversity project - www.caperdu.com - We log our species in iNaturalist.

My goal is to be able to survey the birds that live in the area. My plan was to leave a recording device outside overnight to capture a range of species.
My current workflow is:
1. I create a long recording
2. I airdrop to my mac, I load in Chiperty. 
3. I set the analysis to local birds and a high threshold.
4. The mac is able to crunch several hours of recording easily. 
5. Then I can jump to each interesting 'sighting' and hear the recording
6. Chiperty makes it easy to export that segment and then I load into iNat

So, I guess I would say this is a low priority

Where this workflow is more of a pain is Merlin to iNaturalist on IOS. iNaturalist has a record feature but not a file import on mobile

**Comment by Mattk70** [comment]
Hi David, I had a quick look at the website link, but couldn't find an API. Can you provide a direct link to this so I can investigate? Thanks

**Comment by David-Birdbrain** [comment]
Here is https://api.inaturalist.org/v1/docs/
Im thinking its the identification or observations method

**Comment by Mattk70** [comment]
Thanks, I'll take a look!

**Reply by DD4WH** [reply]
I am not quite sure whether this would be a good idea:
Although most of us only report manually VERIFIED observations to Biodiversity portals, it could be dangerous to open up a possibility for mass reporting of automatically identifed observations (without manual check) directly in Chirpity. 
My opinion is, we should NOT allow direct upload of classified identifications.
No Biodiversity portal wants automatically identified observations! Every portal I know of has been designed for manually identified or manually checked IDs and is dependent on manual checking for accuracy. 
So,
It would be much better to allow only upload of manually checked observations and it is vital that the mass upload of automatically Id'ed classifications is actively prevented. 

Just my two cents...
Keep up the extremely good work on Chirpity!

Frank Dziock DD4WH

**Reply by Mattk70** [reply]
If I was to implement an upload API,  for iNaturalist or anywhere else, there would be guardrails to ensure only those records that had been manually verified would be included in the upload.
